styleguide/content/design-patterns/basic-page-spacing.md

```diff
@@ -0,0 +1,45 @@
+The following page spacing guidelines apply to any custom application that is embedded in a tab inside of Mavenlink, or exists on its own independent page.
+
+<div class="header-four">Base Page Margins</div>
```

Comment:
It might be better to use a heading element to keep accessibility functioning correctly.

In order to provide custom styles to Styleguidist, you will want to read their documentation: See https://react-styleguidist.js.org/docs/cookbook.html#how-to-change-styles-of-a-style-guide

Theoretically, it should be possible to provide custom styles for Heading components. I'm not really sure what they are called but it seems it should be possible to figure it out easily from the generated class name.